meshtastic/tests/test_meshtasticd_tcp_interface_ci.py

```
@@ -0,0 +1,326 @@
+                    raise ValueError(
+                        INVALID_IPV6_BRACKETED_PORT.format(
+                            env_var=MESHTASTICD_HOST_ENV_VAR,
+                            host=host,
+                        )
+                    ) from exc
+            raise ValueError(
+                INVALID_IPV6_BRACKETED_PORT.format(
+                    env_var=MESHTASTICD_HOST_ENV_VAR,
+                    host=host,
+                )
+            )
```

Ruff (0.15.4)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)